### PR TITLE
Add history import password step

### PR DESCRIPTION
**Pull request by daehn**
## What's new in this PR?

* Ask for password when importing a backup.
* Add option to show a subtitle below the loading view.

### CODE REVIEW

File: Cartfile.resolved [hunk 1]
```diff
@@ -12,20 +12,20 @@ github "wireapp/ZipArchive" "v2.0.8"
 github "wireapp/avs-ios-binaries" "4.1.131"
 github "wireapp/ios-snapshot-test-case" "2.1.4-swift3.1"
 github "wireapp/libPhoneNumber-iOS" "0.9.2-swift3.1"
-github "wireapp/mixpanel-swift" "v2.2.2"
+github "wireapp/mixpanel-swift" "v2.3.3"
```
Comment (vytis) [inline Cartfile.resolved]: This is also build for Xcode 9.3

**Comment by typfel** [comment]
@daehn  tests fails to compile due to my voice channel changes. Let me quickly update my UI PR for that..

**Comment by typfel** [comment]
Actually, that won't work.

**Comment by typfel** [comment]
So fixed it here instead :)

**Pull request merged**